lib/appenders/file.js

```diff
@@ -53,6 +53,15 @@ function fileAppender(file, layout, logSize, numBackups, options, timezoneOffset
   let writer = openTheStream(file, logSize, numBackups, options);
 
   const app = function (loggingEvent) {
+    if (options.removeColor === true) {
+      // eslint-disable-next-line no-control-regex
+      const regex = new RegExp("\x1b[[0-9;]*m", "g");
+      for (let i = 0; i < loggingEvent.data.length; i += 1) {
```

Comment:
It would be better to use a map here instead of the loop:
`loggingEvent.data = loggingEvent.data.map(d => d.replace(regex, ''))`